airbyte-webapp/src/core/domain/catalog/traverseSchemaToField.test.ts

```diff
@@ -0,0 +1,106 @@
+import { AirbyteJSONSchema } from "core/jsonSchema";
+
+import { traverseSchemaToField } from "./traverseSchemaToField";
```

Comment:
These are just some basic tests to cover this method, since there were none before.